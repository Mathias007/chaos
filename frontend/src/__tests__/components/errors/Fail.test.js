import React from "react";
import renderer from "react-test-renderer";
import Fail from "../../../pages/errors/Fail";

describe("Fail test", () => {
    it("Fail should match snapshot", () => {
        const component = renderer.create(<Fail />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
