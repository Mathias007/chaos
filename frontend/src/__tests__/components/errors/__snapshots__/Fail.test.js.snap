// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fail test Fail should match snapshot 1`] = `
<div
  className="ant-result ant-result-error"
>
  <div
    className="ant-result-icon"
  >
    <i
      aria-label="icon: close-circle"
      className="anticon anticon-close-circle"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="close-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 0 1-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
        />
      </svg>
    </i>
  </div>
  <div
    className="ant-result-title"
  >
    Submission Failed
  </div>
  <div
    className="ant-result-subtitle"
  >
    Please check and modify the following information before resubmitting.
  </div>
  <div
    className="ant-result-content"
  >
    <div
      className="desc"
    >
      <div
        aria-label={null}
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": null,
          }
        }
      >
        <span
          aria-label={null}
          className="ant-typography"
          style={
            Object {
              "WebkitLineClamp": null,
              "fontSize": 16,
            }
          }
        >
          <strong>
            The content you submitted has the following error:
          </strong>
        </span>
      </div>
      <div
        aria-label={null}
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": null,
          }
        }
      >
        <i
          aria-label="icon: close-circle"
          className="anticon anticon-close-circle"
          style={
            Object {
              "color": "red",
            }
          }
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="close-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M685.4 354.8c0-4.4-3.6-8-8-8l-66 .3L512 465.6l-99.3-118.4-66.1-.3c-4.4 0-8 3.5-8 8 0 1.9.7 3.7 1.9 5.2l130.1 155L340.5 670a8.32 8.32 0 0 0-1.9 5.2c0 4.4 3.6 8 8 8l66.1-.3L512 564.4l99.3 118.4 66 .3c4.4 0 8-3.5 8-8 0-1.9-.7-3.7-1.9-5.2L553.5 515l130.1-155c1.2-1.4 1.8-3.3 1.8-5.2z"
            />
            <path
              d="M512 65C264.6 65 64 265.6 64 513s200.6 448 448 448 448-200.6 448-448S759.4 65 512 65zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
            />
          </svg>
        </i>
         Your account has been frozen
        <a>
          Thaw immediately &gt;
        </a>
      </div>
      <div
        aria-label={null}
        className="ant-typography"
        style={
          Object {
            "WebkitLineClamp": null,
          }
        }
      >
        <i
          aria-label="icon: close-circle"
          className="anticon anticon-close-circle"
          style={
            Object {
              "color": "red",
            }
          }
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="close-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M685.4 354.8c0-4.4-3.6-8-8-8l-66 .3L512 465.6l-99.3-118.4-66.1-.3c-4.4 0-8 3.5-8 8 0 1.9.7 3.7 1.9 5.2l130.1 155L340.5 670a8.32 8.32 0 0 0-1.9 5.2c0 4.4 3.6 8 8 8l66.1-.3L512 564.4l99.3 118.4 66 .3c4.4 0 8-3.5 8-8 0-1.9-.7-3.7-1.9-5.2L553.5 515l130.1-155c1.2-1.4 1.8-3.3 1.8-5.2z"
            />
            <path
              d="M512 65C264.6 65 64 265.6 64 513s200.6 448 448 448 448-200.6 448-448S759.4 65 512 65zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
            />
          </svg>
        </i>
         Your account is not yet eligible to apply
         
        <a>
          Apply Unlock &gt;
        </a>
      </div>
    </div>
  </div>
  <div
    className="ant-result-extra"
  >
    <button
      className="ant-btn ant-btn-primary"
      onClick={[Function]}
      type="button"
    >
      <span>
        Go Console
      </span>
    </button>
    <button
      className="ant-btn"
      onClick={[Function]}
      type="button"
    >
      <span>
        Buy Again
      </span>
    </button>
  </div>
</div>
`;
